scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -108,14 +112,18 @@ def compute_config(args, cpu_flags=cpuflags(), machine=None):
     alldevices = it.chain(it.chain(string.lowercase[1:]),
                           it.product(string.lowercase, string.lowercase))
 
+    configopts = {
+        'nicmodel': 'e1000',
+        'emulator': args.emulator,
+        'vdisk_dir': args.vdiskdir,
+    }
     if hypervisor_has_virtio(libvirt_type):
-        nicmodel = "virtio"
         targetdevprefix = "vd"
-        targetbus = "virtio"
+        configopts['nicmodel'] = 'virtio'
+        configopts['target_bus'] = 'virtio'
     else:
-        nicmodel = "e1000"
```

Comment:
Should this not be populated in the `configopts` dict var? If not then ignore my comment on line 115.